flink-libraries/flink-table/src/test/scala/org/apache/flink/table/api/stream/sql/JoinTest.scala

```diff
@@ -55,12 +55,13 @@ class JoinTest extends TableTestBase {
           unaryNode(
             "DataStreamCalc",
             streamTableNode(1),
-            term("select", "a", "b", "proctime")
+            term("select", "a", "b", "-(proctime, 3600000) AS -",
```

Comment:
I think this issue may be caused by CALCITE-1753, still investigating the changes in that JIRA.